src/main/java/de/tum/in/www1/artemis/web/rest/AssessmentResource.java

```diff
@@ -177,11 +181,24 @@ protected ResponseEntity<Result> getExampleAssessment(long exerciseId, long subm
         // if the user is a tutor and the submission is not used for tutorial in the assessment dashboard
         // The reason is that example submissions with isTutorial = false should be shown immediately (with the assessment) to the tutor and
         // for example submission with isTutorial = true, the assessment should not be shown to the tutor. Instead, the tutor should try to assess it him/herself
-        if (!(isAtLeastEditor || (isAtLeastTutor && !exampleSubmission.isUsedForTutorial()))) {
-            throw new AccessForbiddenException();
+        // Therefore we send a result with only the references included, which is needed to tell the tutor which elements he missed to assess
+        Result result = assessmentService.getExampleAssessment(submissionId);
+
+        if (!Objects.isNull(result) && !(isAtLeastEditor || (isAtLeastTutor && !exampleSubmission.isUsedForTutorial()))) {
+            Result freshResult = new Result();
+            freshResult.setId(result.getId());
+            if (Hibernate.isInitialized(result.getFeedbacks())) {
```

Comment:
Shouldn't we make sure that it is initialized, since returning the empty result will not help the tutor?
So you might want to add an else branch where you load this feedback or make sure that getExampleAssessment always returns the result with its feedbacks

Reply:
That was just an extra safety measure. I looked at the db call and it is always initialized. So I removed the check in 2af7ba628106f8e36de41d3e1a646d2b3faa3d71